test/python/aievec.py

```diff
@@ -46,39 +46,23 @@ def demo_fun1():
 @construct_and_print_module
 def test_aievec():
     @func
-    def mul_mul(
-        A: T.memref(2048, T.f32()),
-        B: T.memref(2048, T.f32()),
-        C: T.memref(2048, T.f32()),
-        d: T.f32(),
+    def mul_elem(
+        A: T.memref(2048, T.i16()),
+        B: T.memref(2048, T.i16()),
+        C: T.memref(2048, T.i16()),
     ):
-        v0 = vector.broadcast(T.vector(8, T.f32()), d)
-        v1 = aievec.concat([v0, v0])
-        for i in scf.for_(0, 2048, 8):
-            v2 = aievec.upd(T.vector(8, T.f32()), A, [i])
-            v3 = aievec.upd(T.vector(8, T.f32()), B, [i])
-            v4 = aievec.mul(
-                T.vector(8, T.f32()),
+        for i in scf.for_(0, 2048, 32):
+            v0 = aievec.upd(T.vector(32, T.i16()), A, [i])
+            v1 = aievec.upd(T.vector(32, T.i16()), B, [i])
+            v2 = aievec.mul_elem(
```

Comment:
The original test was targeting AIE1.
Are the aievec.aie1 ops no longer accessible through the python interface?

Reply:
Yes, right. The `AIEVecAIE1Dialect` is not exposed to the Python interface.